plugins/scaffolder/src/components/fields/RepoBranchPicker/RepoBranchPicker.tsx

```diff
@@ -127,10 +138,7 @@ export const RepoBranchPicker = (props: RepoBranchPickerProps) => {
             onChange={updateLocalState}
             state={state}
             rawErrors={rawErrors}
-            accessToken={
-              uiSchema?.['ui:options']?.requestUserCredentials?.secretsKey &&
-              secrets[uiSchema['ui:options'].requestUserCredentials.secretsKey]
-            }
+            accessToken={getAccessToken()}
```

Comment:
I mentioned in a previous PR that I think that we can clean this up and push down the token collection into those components and just duplicate it rather than passing it through. It's a little unfortunate to have to pick them out like this I think.

Reply:
Maybe we can leave this for now like you've done though, and revisit in a separate refactor PR.